widget/info.json

```diff
@@ -6,7 +6,7 @@
         "certified": "Yes",
         "publisher": "Fortinet",
         "snapshots": [
-            "https:\/\/repo.fortisoar.fortinet.com/\/widgets\/socManagement-1.0.0\/images\/soc-management.png"
+            "https://repo.fortisoar.fortinet.com/fsr-widgets/socManagement-1.0.0/images/soc-management.png"
```

Comment:
Correct the URL https://repo.fortisoar.fortinet.com/content-hub/socManagement-1.0.0/latest/images/soc-management.png